install/gcp/scripts/run_forseti.sh

```diff
@@ -21,6 +21,10 @@ source /home/ubuntu/forseti_env.sh
 # they will be logged.
 set -x
 
+# Remove any CAI temp files generated by the forseti process.
+# Failed inventory due to various reasons could cause the temp files not being deleted.
+sudo rm -rf /tmp/forseti-cai-*
```

Comment:
Are there other /tmp files that could be in here that we would want to remove?

Reply:
I dont think there are any, these are the newly generated sqlite files only for the new branch.